client/landing/subscriptions/components/site-list/site-row.tsx

```diff
@@ -57,7 +58,10 @@ export default function SiteRow( {
 				<span className="title-box" role="cell">
 					{ siteIcon }
 					<span className="title-column">
-						<span className="name">{ name }</span>
+						<span className="name">
+							{ name }
+							{ organization_id && <span className="p2-label">P2</span> }
```

Comment:
For `organization_id === 0`, the statement will render `"0"` string instead of `null`:

![Markup on 2023-04-26 at 14:30:40](https://user-images.githubusercontent.com/25105483/234575191-5bf3b11e-bcd4-4398-9794-f7c7c54d2185.png)

Depending on how the final logic will look like, we may want to use `? <span></span> : null` instead.

Reply:
Thanks for pointing this out! In fact, making double not (`!! organization_id`) fixes the problem.